src/entity/Living.php

```diff
@@ -329,7 +329,11 @@ public function fall(float $fallDistance) : void{
 			}
 		}
 
-		$fallBlock->onEntityFall($this, $fallDistance, $fallBlockPos);
+		if ($fallBlock instanceof Flowable ){
+			$fallBlockPos = $fallBlockPos->subtract(0, 1, 0);
+			$fallBlock = $this->getWorld()->getBlock($fallBlockPos);
+		}
+		$fallBlock->onEntityFall($this, $fallDistance, $this->getJumpVelocity());
```

Comment:
This isn't useful. The actual velocity of the impact is what I was talking about.

Reply:
Is that the damage?

Reply:
No, it's the velocity. (or motion, since PM likes to incorrectly name things.)

Reply:
In fact, since it's unrelated to the PR, you can omit it entirely.